pkg/kubelet/cm/memorymanager/memory_manager_test.go

```diff
@@ -213,3 +298,2109 @@ func TestConvertPreReserved(t *testing.T) {
 		})
 	}
 }
+
+func TestGetSystemReservedMemory(t *testing.T) {
+	machineInfo := cadvisorapi.MachineInfo{
```

Comment:
I can see that you use the same machine info topology under tests, can you move it under to the var under the file and re-use it. It can save some lines

Reply:
I've extracted it to separate function: `returnMachineInfo`. Similar approach was used in TM Scope testing.